lib/rails-settings-ui/version.rb

```diff
@@ -1,3 +1,3 @@
 module RailsSettingsUi
-  VERSION = '1.3.1'
+  VERSION = '1.3.2'
```

Comment:
please add changes to `CHANGES.md` and change `inline_main_app_routes!` method in README + add description of the new config attribute `engine_name` to Config section of README

Reply:
@accessd updated

Reply:
please add description of your contribution to CHANGES.md

Reply:
@accessd i did :nerd_face: